webapp/packages/plugin-connection-template/src/ConnectionDialog/TemplateConnectionSelector/TemplateConnectionSelector.tsx

```diff
@@ -8,24 +8,27 @@
 import { observer } from 'mobx-react-lite';
 import { useMemo, useState } from 'react';
 
-import { ItemList, ItemListSearch } from '@cloudbeaver/core-blocks';
-import type { Connection, DBDriver } from '@cloudbeaver/core-connections';
+import { ItemList, ItemListSearch, useResource } from '@cloudbeaver/core-blocks';
+import { DBDriverResource } from '@cloudbeaver/core-connections';
+import { useService } from '@cloudbeaver/core-di';
+import { CachedMapAllKey } from '@cloudbeaver/core-resource';
 
+import { TemplateConnectionsResource } from '../../TemplateConnectionsResource';
+import { TemplateConnectionsService } from '../../TemplateConnectionsService';
 import { TemplateConnectionItem } from './TemplateConnectionItem';
 
 interface Props {
-  templateConnections: Connection[];
-  dbDrivers: Map<string, DBDriver>;
   className?: string;
   onSelect: (dbSourceId: string) => void;
 }
 
-export const TemplateConnectionSelector = observer<Props>(function TemplateConnectionSelector({
-  templateConnections,
-  dbDrivers,
-  className,
-  onSelect,
-}) {
+export const TemplateConnectionSelector = observer<Props>(function TemplateConnectionSelector({ className, onSelect }) {
+  useResource(TemplateConnectionSelector, TemplateConnectionsResource, undefined);
```

Comment:
please also load all projects from ProjectInfoResource (you need it to access projectTemplates and also add `{ forceSuspense: true }` to useResource without data access (to trigger suspense)